app/web/src/components/EditorMenu/menuConfig.tsx

```diff
@@ -5,98 +5,104 @@ import { useSaveCode } from 'src/components/IdeWrapper/useSaveCode'
-                
-                return <DropdownItem {...props} />
-            },
-        },
-        {
-            label: 'Download STL',
-            shortcut: 'ctrl+shift+d, command+shift+d',
-            shortcutLabel: cmdOrCtrl() + ' Shift D',
-            component: (props) => {
-                const { state, thunkDispatch, config } = props
-                const handleStlDownload = makeStlDownloadHandler({
-                    type: state.objectData?.type,
-                    ideType: state.ideType,
-                    geometry: state.objectData?.data,
-                    quality: state.objectData?.quality,
-                    fileName: PullTitleFromFirstLine(state.code || ''),
-                    thunkDispatch,
-                })
+        config.callback = onRender
 
-                config.callback = handleStlDownload
+        return <DropdownItem {...props} />
+      },
+    },
+    {
+      label: 'Download STL',
+      shortcut: 'ctrl+shift+d, command+shift+d',
+      shortcutLabel: cmdOrCtrl() + ' Shift D',
+      component: (props) => {
+        const { state, thunkDispatch, config } = props
+        const handleStlDownload = makeStlDownloadHandler({
+          type: state.objectData?.type,
+          ideType: state.ideType,
+          geometry: state.objectData?.data,
+          quality: state.objectData?.quality,
+          fileName: PullTitleFromFirstLine(state.code || ''),
+          thunkDispatch,
+        })
 
-                return <DropdownItem {...props} />
-            }
-        }
-    ]
+        config.callback = handleStlDownload
+
+        return <DropdownItem {...props} />
+      },
+    },
+  ],
 }
 
 const editMenuConfig = {
-    name: 'edit',
-    label: 'Edit',
-    disabled: true,
-    items: [],
+  name: 'edit',
+  label: 'Edit',
+  disabled: true,
+  items: [],
 }
 
 const viewMenuConfig = {
-    name: 'view',
-    label: 'View',
-    disabled: false,
-    items: [
-        {
-            label: 'Reset layout',
-            shortcut: 'ctrl+alt+r, command+alt+r',
-            shortcutLabel: cmdOrCtrl() + ' Alt R',
-            component: (props) => {
-                const { config, thunkDispatch } = props
-                config.callback = () => thunkDispatch({ type: 'resetLayout' })
-                return <DropdownItem {...props } />
-            }
-        },
-    ],
+  name: 'view',
+  label: 'View',
+  disabled: false,
+  items: [
+    {
+      label: 'Reset layout',
+      shortcut: 'ctrl+shift+r',
+      shortcutLabel: 'Ctrl Shift R',
+      component: (props) => {
+        const { config, thunkDispatch } = props
+        config.callback = () => thunkDispatch({ type: 'resetLayout' })
+        return <DropdownItem {...props} />
+      },
+    },
+    {
+      label: 'All shortcuts',
+      shortcut: 'ctrl+shift+/',
+      shortcutLabel: 'Ctrl Shift /',
+      component: (props) => {
+        const { config } = props
+        const [open, setOpen] = useShortcutModalContext()
```

Comment:
This won't work currently because I backed out of writing it, but @Irev-Dev should I use a React Context to make the `AllShortcutsModal` able to be opened from outside of the component? I could make its own little context to provide around the `IdeWrapper` component, or the page component itself. I might try that and see if you like it.

Reply:
Tried this approach below.